test/karma/detected.js

```diff
@@ -29,6 +29,7 @@ module.exports = function(config) {
         postDetection: function(availableBrowsers) {
           var safariIndex = availableBrowsers.indexOf('Safari');
           if(safariIndex !== -1) {
+            console.log("Not running safari it is/was broken");
```

Comment:
The old unit tests only ran against Chome/Firefox and when I attempted to run them in safari they all failed. Once I updated the code I tried to run them again and they failed, so I turned off safari. Safari does launch and run, but a lot of the unit tests fail.